README.md

```diff
@@ -37,7 +37,6 @@ There are two ways to prove the program:
 - [Go : 1.22.1](https://go.dev/dl)
 - [Rust: 1.81.0-nightly](https://www.rust-lang.org/tools/install)
 - Hardware: X86 CPU, 32 cores, 32G memory
```

Comment:
can you also plz help remove this requirement?  actually the golang and rust can be upgradable, and the hardware is also not a must now.